src/deserialize_null_array.cpp

```diff
@@ -0,0 +1,43 @@
+#include "sparrow_ipc/deserialize_null_array.hpp"
+
+#include <unordered_set>
+#include <sparrow/null_array.hpp>
+
+#include "sparrow_ipc/arrow_interface/arrow_array.hpp"
+#include "sparrow_ipc/arrow_interface/arrow_schema.hpp"
+
+namespace sparrow_ipc
+{
+    sparrow::null_array deserialize_non_owning_null(
+        const org::apache::arrow::flatbuf::RecordBatch& record_batch,
+        std::span<const uint8_t> body,
+        std::string_view name,
+        const std::optional<std::vector<sparrow::metadata_pair>>& metadata,
+        bool nullable,
+        size_t& buffer_index
+    )
+    {
+        constexpr std::string_view format = "n";
```

Comment:
Can we use something like this to avoid hardcoding it and be consistent with the rest of similar functions?
```suggestion
        const std::string_view format = data_type_to_format(
            sparrow::detail::get_data_type_from_array<sparrow::null_array>::get()
        );
```